### ISSUE TITLE
Variable `productId` does not exist

### ISSUE THREAD

**Issue by chrismlusk** [servdhost/craft-asset-storage#28]
Updated to 2.4.4 and am now unable to edit entries in the CP.

```
Twig\Error\RuntimeError: Variable "productId" does not exist. in /var/www/html/vendor/servd/craft-asset-storage/src/templates/cp-extensions/static-cache-clear.twig:1
Stack trace:
#0 /var/www/html/storage/runtime/compiled_templates/c5/c5f824e125354712a180579eb9df349cc5e25f9292ca9e1b4881df299589e27a.php(38): __TwigTemplate_3c6876afa69874fd3cc05dcfcdb51b72842e16ca2b2a15cee36647fc7759f957->{closure}()
#1 /var/www/html/vendor/twig/twig/src/Template.php(405): __TwigTemplate_3c6876afa69874fd3cc05dcfcdb51b72842e16ca2b2a15cee36647fc7759f957->doDisplay(Array, Array)
#2 /var/www/html/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#3 /var/www/html/vendor/twig/twig/src/Template.php(390): Twig\Template->display(Array)
#4 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render(Array, Array)
#5 /var/www/html/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render(Array)
#6 /var/www/html/vendor/craftcms/cms/src/web/View.php(408): Twig\Environment->render('servd-asset-sto...', Array)
#7 /var/www/html/vendor/servd/craft-asset-storage/src/StaticCache/StaticCache.php(495): craft\web\View->renderTemplate('servd-asset-sto...', Array)
#8 /var/www/html/vendor/craftcms/cms/src/web/View.php(1659): servd\AssetStorage\StaticCache\StaticCache->servd\AssetStorage\StaticCache\{closure}(Array, false)
#9 /var/www/html/storage/runtime/compiled_templates/57/57558496936c7b1be9c4434e88ac06367320a129dd9b219bebc168cd9a240d69.php(285): craft\web\View->invokeHook('cp.entries.edit...', Array)
#10 /var/www/html/vendor/twig/twig/src/Template.php(182): __TwigTemplate_709fd5cbf7ceb36cd25a04f099f5c2c51b39436a32c7f1cb2fd390c7e276e616->block_details(Array, Array)
#11 /var/www/html/vendor/twig/twig/src/Template.php(254): Twig\Template->displayBlock('details', Array, Array, true)
#12 /var/www/html/storage/runtime/compiled_templates/8a/8a9e0214e5928132535717a70d5d8df1ee2e5312200194b684a092faaacfde1e.php(101): Twig\Template->renderBlock('details', Array, Array)
#13 /var/www/html/vendor/twig/twig/src/Template.php(405): __TwigTemplate_a1c472f3e3fdc9ef2d671ed905b43c78b1cbf434684a8b2c764092db9f7837c5->doDisplay(Array, Array)
#14 /var/www/html/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#15 /var/www/html/storage/runtime/compiled_templates/f5/f53bf3d59a7b94c60f35defad2e161462ad2a988652f144b36d4ad727e736dd1.php(342): Twig\Template->display(Array, Array)
#16 /var/www/html/vendor/twig/twig/src/Template.php(405): __TwigTemplate_75ec7ec0ecf9a7e0943bf54e10a5409aa01410378d54fdd230b3b78857cf8d39->doDisplay(Array, Array)
#17 /var/www/html/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#18 /var/www/html/storage/runtime/compiled_templates/57/57558496936c7b1be9c4434e88ac06367320a129dd9b219bebc168cd9a240d69.php(124): Twig\Template->display(Array, Array)
#19 /var/www/html/vendor/twig/twig/src/Template.php(405): __TwigTemplate_709fd5cbf7ceb36cd25a04f099f5c2c51b39436a32c7f1cb2fd390c7e276e616->doDisplay(Array, Array)
#20 /var/www/html/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#21 /var/www/html/vendor/twig/twig/src/Template.php(390): Twig\Template->display(Array)
#22 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render(Array, Array)
#23 /var/www/html/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render(Array)
#24 /var/www/html/vendor/craftcms/cms/src/web/View.php(408): Twig\Environment->render('entries/_edit', Array)
#25 /var/www/html/vendor/craftcms/cms/src/web/View.php(461): craft\web\View->renderTemplate('entries/_edit', Array)
#26 /var/www/html/vendor/craftcms/cms/src/web/Controller.php(257): craft\web\View->renderPageTemplate('entries/_edit', Array, 'cp')
#27 /var/www/html/vendor/craftcms/cms/src/controllers/EntriesController.php(248): craft\web\Controller->renderTemplate('entries/_edit', Array)
#28 [internal function]: craft\controllers\EntriesController->actionEditEntry(Object(craft\models\Section), 501512, NULL, NULL, Object(craft\models\Site), Object(craft\elements\Entry))
#29 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#30 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#31 /var/www/html/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction('edit-entry', Array)
#32 /var/www/html/vendor/yiisoft/yii2/base/Module.php(534): craft\web\Controller->runAction('edit-entry', Array)
#33 /var/www/html/vendor/craftcms/cms/src/web/Application.php(277): yii\base\Module->runAction('entries/edit-en...', Array)
#34 /var/www/html/vendor/yiisoft/yii2/web/Application.php(104): craft\web\Application->runAction('entries/edit-en...', Array)
#35 /var/www/html/vendor/craftcms/cms/src/web/Application.php(262): yii\web\Application->handleRequest(Object(craft\web\Request))
#36 /var/www/html/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#37 /var/www/html/web/index.php(22): yii\base\Application->run()
#38 {main}
```

**Comment by mattgrayisok** [comment]
Sorry about that! Fix just released in 2.4.5.

Any other issues, let me know 👍

**Comment by chrismlusk** [comment]
Thanks!
